lib/honeykiq/server_middleware.rb

```diff
@@ -18,9 +18,19 @@ def call(_worker, msg, queue_name)
       event&.send
     end
 
+    def extra_fields
+      {}
+    end
+
     private
 
-    def all_fields(msg, queue_name)
+    def run_with_hooks(event, msg, queue_name)
```

Comment:
not sure about `_hooks` 🤔 but it a minute point on a private method! 🚲 🏡

Reply:
🤔